wls-common/pom.xml

```
@@ -23,7 +23,7 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <maven.compiler.release>${java.version}</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <spring.boot.version>3.3.5</spring.boot.version>
+        <spring.boot.version>3.4.3</spring.boot.version>
         <spring.cloud.version>2023.0.5</spring.cloud.version>
```

https://spring.io/projects/spring-cloud

```suggestion
        <spring.cloud.version>2024.0.0</spring.cloud.version>
```